daemon/k8s_watcher_test.go

```diff
@@ -103,11 +103,11 @@ func (ds *DaemonSuite) Test_missingK8sNetworkPolicyV1(c *C) {
 			name: "repository contains all policies",
 			setupArgs: func() args {
 				p1 := policy.NewPolicyRepository()
-				_, err := p1.Add(api.Rule{
+				_, _, err := p1.Add(api.Rule{
 					EndpointSelector: api.NewESFromLabels(labels.ParseSelectLabel("id=a")),
 					Labels: labels.LabelArray{labels.NewLabel(k8sConst.PolicyLabelName, "bar", labels.LabelSourceK8s),
 						labels.NewLabel(k8sConst.PolicyLabelNamespace, "foo", labels.LabelSourceK8s)},
-				})
+				}, map[uint16]*identity.Identity{})
```

Comment:
Make this a type and and add a utility function to construct a new instance of it.